app/controllers/admin/qualification_report_uploads_controller.rb

```diff
@@ -0,0 +1,17 @@
+module Admin
+  class QualificationReportUploadsController < BaseAdminController
+    before_action :ensure_service_operator
+
+    def new
+    end
+
+    def create
+      @dqt_report_consumer = AutomatedChecks::DQTReportConsumer.new(params[:file], admin_user)
+      if @dqt_report_consumer.ingest
```

Comment:
If we're careful to avoid N+1s and how we read the CSV I think we can avoid the complexity of moving this to a background job. The action is idempotent and a fast response isn't essential given the frequency this is done and the fact that it's for internal users.